doc/man3/X509_STORE_CTX_get_error.pod

```diff
@@ -346,7 +350,8 @@ Unhandled critical CRL extension.
 
 =item B<X509_V_ERR_INVALID_NON_CA: invalid non-CA certificate (has CA markings)>
 
-Invalid non-CA certificate has CA markings.
+
+Non-CA certificate required but has CA markings.
```

Comment:
```suggestion
A CA certificate was encountered in a context where only a non-CA certificates are expected.
```

Reply:
Changed to: CA certificate was encountered where only a non-CA certificate is expected.